### ISSUE TITLE
Build failure due to ambiguous occurrence

### ISSUE THREAD

**Issue by snoyberg** [juhp/dl-fedora#2]
Looks to be caused by http-directory 0.1.9 release

```
/var/stackage/work/unpack-dir/unpacked/dl-fedora-0.9.2-91d27ab1ffca5c0b05de962f906a3edf442585ce3d0a9bed3f945177f02ad825/src/Main.hs:185:28: error:                                                                   Ambiguous occurrence ‘+/+’                                                                                                                                                                                       It could refer to                                                                                                                                                                                                   either ‘Network.HTTP.Directory.+/+’,                                                                                                                                                                                    imported from ‘Network.HTTP.Directory’ at src/Main.hs:24:1-29                                                                                                                                                 or ‘Main.+/+’, defined at src/Main.hs:505:4                                                                                                                                                               |                                                                                                                                                                                                            185 |                _ -> dlFpo) +/+ path <> "/"                                                                                                                                                                     |                            ^^^

/var/stackage/work/unpack-dir/unpacked/dl-fedora-0.9.2-91d27ab1ffca5c0b05de962f906a3edf442585ce3d0a9bed3f945177f02ad825/src/Main.hs:198:37: error:
    Ambiguous occurrence ‘+/+’
    It could refer to
       either ‘Network.HTTP.Directory.+/+’,
              imported from ‘Network.HTTP.Directory’ at src/Main.hs:24:1-29
           or ‘Main.+/+’, defined at src/Main.hs:505:4
    |
198 |               masterUrl = masterDir +/+ file
    |                                     ^^^

/var/stackage/work/unpack-dir/unpacked/dl-fedora-0.9.2-91d27ab1ffca5c0b05de962f906a3edf442585ce3d0a9bed3f945177f02ad825/src/Main.hs:217:71: error:
    Ambiguous occurrence ‘+/+’
    It could refer to
       either ‘Network.HTTP.Directory.+/+’,
              imported from ‘Network.HTTP.Directory’ at src/Main.hs:24:1-29
           or ‘Main.+/+’, defined at src/Main.hs:505:4
    |
217 |           return (finalurl, prefix, (masterUrl,masterSize), (finalDir +/+) . T.unpack <$> mchecksum, already)
    |                                                                       ^^^

/var/stackage/work/unpack-dir/unpacked/dl-fedora-0.9.2-91d27ab1ffca5c0b05de962f906a3edf442585ce3d0a9bed3f945177f02ad825/src/Main.hs:232:63: error:
    Ambiguous occurrence ‘+/+’
    It could refer to
       either ‘Network.HTTP.Directory.+/+’,
              imported from ‘Network.HTTP.Directory’ at src/Main.hs:24:1-29
           or ‘Main.+/+’, defined at src/Main.hs:505:4
    |
232 |                 if mirror /= downloadFpo then return $ mirror +/+ path +/+ file
    |
```

**Comment by juhp** [comment]
Oh yes thanks

**Comment by juhp** [comment]
I believe this was fixed in 0.9.3 released (released this month)
